### PR TITLE
project setup

### PR DESCRIPTION
**Pull request by Danny-imo**
### **In This Branch:**

- [ ] Added Linters
- [ ] Added HTML
- [ ] Added CSS

### CODE REVIEW

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi <STUDENT_MENTION>, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- No linter errors :+1: 
- Good use of GitHub flow :+1: 
- Descriptive PR and commit messages :+1: 


## Required Changes ♻️
- [ REQUIRED] Kindly make sure your README file is customized with your own data considering only the FAQ section optional and removing/replacing the Microverse logo :recycle: 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 1]
```diff
@@ -58,69 +35,20 @@ After you're finished please remove all the comments and instructions!
...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-> Add a link to your deployed project.
 
 - [Live Demo Link](https://google.com)
```
Comment (braincee) [inline README.md]: - [ ] [OPTIONAL] I kindly suggest to remove the live demo link since the project has not been deployed yet

File: README.md [hunk 2]
```diff
@@ -58,69 +35,20 @@ After you're finished please remove all the comments and instructions!
...
 
-<!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-> Describe how a new developer could make use of your project.
-
 To get a local copy up and running, follow these steps.
 
 ### Prerequisites
```
Comment (braincee) [inline README.md:48]: - [ ] [OPTIONAL] I kindly suggest adding some prerequisites to the README file to look professional
Example:
- Have some knowledge of HTML and CSS
- Be familiar with git commands
- Basics understanding of Linters

**Review by braincee — CHANGES_REQUESTED** [review]
Hello @Danny-imo  ✋, I am @braincee your Code Reviewer

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


so-close

HIGHLIGHTS
------

 - GitHub flow rightfully used ✅
 - Meaningful title and summary for PR ✅
 - Professional README.md file ✅
 - No linter errors ✅

REQUIRED CHANGES ♻️
------

- [ ] Please ensure that you add setup instructions to the README file for other developers to clone the repository. Example: `https://github.com/Danny-imo/portfolio.git`
- [ ] Kindly update the MIT with a link to the MIT file in your README file.
- [ ] KIndly add a specific installation process in the README file. Example `npm install`.
- [ ] KIndly add social media links to the author section. Example Twitter, LinkedIn and Facebook

**_Please check the comments under the review._**

OPTIONAL CHANGES
--------
**_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @braincee in your question so I can receive the notification.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

----

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -58,172 +32,74 @@ After you're finished please remove all the comments and instructions!
...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FUTURE FEATURES -->
 
 ## 🔭 Future Features <a name="future-features"></a>
 
-> Describe 1 - 3 features you will add to the project.
 
 - [ ] **[new_feature_1]**
 - [ ] **[new_feature_2]**
```
Comment (Shedrack-Sunday) [inline README.md:105]: - [Optional] Please add three future features to this section of the README to show possibilities of possible features in the future 👍

File: README.md [hunk 4]
```diff
@@ -281,7 +154,7 @@ I would like to thank...
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](./MIT.md) licensed.
 
 _NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (Shedrack-Sunday) [inline README.md:159]: - [Optional] This sectio was meant to help and give a guide. Kindly delete it

**Review by Shedrack-Sunday — APPROVED** [review]
# STATUS: APPROVED ♻️♻️
Hi @Danny-imo 

To highlight 👍🏽
✔️ Proper use of .gitignore
✔️ No linter errors

Your project is complete! There is nothing else to say other than... it's time to merge it 🚀
Congratulations! 🎉

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**Pull request merged**